<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QListView" name="contactlist"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="toolbar">
        <item>
         <widget class="QPushButton" name="addButton">
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteButton">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="saveButton">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="searchButton">
          <property name="text">
           <string>Search</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QFrame" name="dataFrame">
      <layout class="QVBoxLayout" name="datalayout">
       <item>
        <widget class="QGroupBox" name="IdentieferBox">
         <property name="title">
          <string>Identiefer</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLineEdit" name="fnameEdit">
            <property name="placeholderText">
             <string>First name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lnameEdit">
            <property name="placeholderText">
             <string>Last name</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="adressBox">
         <property name="title">
          <string>Address</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLineEdit" name="countryEdit">
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>Country</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="cityEdit">
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>City</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="phonesBox">
         <property name="title">
          <string>Phones</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QPlainTextEdit" name="numlistWidget">
            <property name="placeholderText">
             <string>Type num</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Email</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLineEdit" name="emailEdit"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
